src/classes/routes/RouterFactory.ts

```diff
@@ -0,0 +1,60 @@
+import { Router, Request, Response, NextFunction } from 'express';
+import PostImageController from './PostImageController';
+import Logger from '../Logger';
+import ConfigurationManager from '../ConfigurationManager';
+import * as multer from 'multer';
+import * as cors from 'cors';
+import { ExpressMiddleware } from '../../types';
+
+class RouterFactory {
+
+    constructor (private configurationManager: ConfigurationManager, private logger: Logger) { }
+
+    public buildImageRouter (): Router {
+        const router: Router = Router();
+        const fileInputName = this.configurationManager.getFileInputName();
+        const uploadFilePath = this.configurationManager.getUploadPath();
+
+        router.use(this.createCorsMiddleware());
+
+        router.options('*', this.createCorsMiddleware());
+
+        router.post('/image', this.createUploadMiddleware(fileInputName, uploadFilePath), (request: Request, response: Response, next: NextFunction) => {
+            const controller = new PostImageController(this.logger);
+            controller.executeController(request, response, next);
+        });
```

Comment:
As my understanding, after posting an image into the server it will return a response with that image, hmm no method for getting images by id or name?